broker/src/main/java/org/apache/rocketmq/broker/topic/TopicConfigManager.java

```diff
@@ -124,6 +124,14 @@ public TopicConfigManager(BrokerController brokerController) {
             topicConfig.setWriteQueueNums(1);
             this.topicConfigTable.put(topicConfig.getTopicName(), topicConfig);
         }
+        {
+            String topic = MixAll.SCHEDULE_TOPIC;
+            TopicConfig topicConfig = new TopicConfig(topic);
```

Comment:
SCHEDULE_TOPIC is a special system topic? IMO, we should refactor this naming logic here.

Reply:
Yeah, it is a special system topic used in delay msg. When producer send a delay  msg, the msg will be send to SCHEDULE_TOPIC, then send to the real topic when the delay time comes.
Do u meaning  add prefix to system topic , or remove system topics to a independent class?

Reply:
system topic should be a List, also, we could judge it from the naming, a prefix is a good way.